app/controllers/state_file/questions/unemployment_controller.rb

```diff
@@ -10,14 +10,18 @@ def self.show?(intake)
       end
 
       def self.navigation_actions
-        [:new, :index]
+        [:index, :new]
       end
 
       def index
         @state_file1099_gs = current_intake.state_file1099_gs
+        unless @state_file1099_gs.present?
+          redirect_to action: :new
+        end
```

Comment:
If there is no 1099G's present, we show the form that asks users whether they want to create one